### ISSUE TITLE
"Open recent" menu is buggy. Flickers when moused over.

### ISSUE THREAD

**Issue by shivendra90** [spyder-ide/spyder-notebook#215]
## Description of your problem
When trying to open a recent notebook through clicking on the gear button on top right side of the 'Notebook' pane, hovering over the "Open recent" section should display recent notebooks as usual but the menu starts ~~flashing~~ flickering and it becomes undecipherable.

**What steps will reproduce the problem?**

1. Open a Spyder session.
2. With the Notebook pane open, click on the "options" button which is a small gear on top right side.
3. Mouse over "Open recent" section.

**What is the expected output? What do you see instead?**
Clear menu should be displayed but the menu starts ~~flashing~~ flickering.

**Please provide any additional information below (stack trace, screenshots, a gif, etc)**

## Versions and main components

* Spyder Version: 3.3.6
* Python Version: 3.7.0
* Qt Version: 5.9.7
* PyQt Version: 5.9.2
* Jupyter Notebook Version: 6.0.0
* Operating system: macOS Mojave 10.14.2

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:
IPython >=4.0     :  7.7.0 (OK)
cython >=0.21     :  0.29.13 (OK)
jedi >=0.9.0      :  0.15.1 (OK)
matplotlib >=2.0.0:  3.1.1 (OK)
nbconvert >=4.0   :  5.5.0 (OK)
numpy >=1.7       :  1.16.4 (OK)
pandas >=0.13.1   :  0.25.0 (OK)
psutil >=0.3      :  5.6.3 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pyflakes >=0.6.0  :  2.1.1 (OK)
pygments >=2.0    :  2.4.2 (OK)
pylint >=0.25     :  2.3.1 (OK)
qtconsole >=4.2.0 :  4.5.3 (OK)
rope >=0.9.4      :  0.14.0 (OK)
sphinx >=0.6.6    :  2.1.2 (OK)
sympy >=0.7.3     :  1.4 (OK)

**Comment by ccordoba12** [comment]
> but the menu starts flashing and it becomes undecipherable.

Please post a screenshot or a little video to understand what you mean by this.

**Comment by shivendra90** [comment]
I'm sorry about the terminology used; maybe 'flickering' should be the right word here. I do not have any video recording software so I won't be able to upload anything. But what I meant to say that once you hover the mouse to "Open recent" section, the menu starts flickering and it becomes unreadable the files it shows.

**Comment by ccordoba12** [comment]
Did you open notebooks with very long names?

**Comment by shivendra90** [comment]
The notebook I'm trying to open is called `eli5_model_diagnosis.ipynb` which I think is a long name. I renamed it to `eli5.ipynb` though but the menu still flickers.

**Comment by jitseniesen** [comment]
I think this is caused by the `self.options_menu.clear()` in `update_notebook_actions()`, which is called when the `Open recent` menu is about to be shown. This clears the options menu which is at that time being displayed.
